public_html/wp-content/mu-plugins/blocks/assets/src/shared/i18n/index.js

```diff
@@ -0,0 +1,167 @@
+/**
```

Comment:
All commit headings are now < 75 chars

Reply:
I think they need to stop at 72, otherwise GH will wrap. You can see that this one is still wrapping:

`Blocks: Include newly added Sponsor, Session and Organizer block support.`

...because it's 73.

[Core actually says to stopping at 70](https://make.wordpress.org/core/handbook/best-practices/commit-messages/), but I think that's somewhat arbitrary, since Trac will display up to 80 IIRC. Stopping at 72 should make us compatible w/ GH and Trac.